js/modules/swiperHelpers.js

```diff
@@ -0,0 +1,35 @@
+export function setSlidesState(slider, className){
+	const activeIndexes = getActiveIndexes(slider, slider.realIndex);
+	const slideCollection = slider.slides;
+	
+	slideCollection.forEach(function(elem, index) {
+		if(activeIndexes.includes(index)){
+			elem.classList.remove(className);
+		}
+		else{
+			elem.classList.add(className);
+		}
+	});
+}
+
+export function getActiveIndexes(slider, realIndex){
```

Comment:
do you need this function somewhere else?
if not, maybe you don't need to export this function?

Reply:
Done.